lib/pipeline.js

```diff
@@ -37,28 +35,6 @@ class PipelineModel extends BaseModel {
     }
 
     /**
-     * Format the scm url to include a branch and make case insensitive
-     * @method formatScmUrl
-     * @param  {String}     scmUrl Github scm url
-     * @return {String}            Lowercase scm url with branch name
-     */
-    formatScmUrl(scmUrl) {
```

Comment:
Yeah, it was used statically in one place in the api in a place where it makes no sense to instantiate a new pipeline just to use this function. I moved the function to the appropriate place in the API. If it REALLY needs to be used elsewhere, we can make a helper module for it.